tests/scripts/expect/tun-dns-client.exp

```diff
@@ -54,4 +54,10 @@ send "dns resolve4 ipv6.google.com ::1 53\n"
 expect "DNS response for ipv6.google.com"
 expect_line "Done"
 
+send "dns resolve ipv6.google.com ::1 53 6000 4 1 tcp\n"
+expect "DNS response for ipv6.google.com"
+expect_line "Done"
+
+send ""
+
```

Comment:
@jwhui Will this be sufficient as tests or there is something extra to implement? I will appreciate a hint.